src/app/ClusterObjectUtils.h

```diff
@@ -0,0 +1,312 @@
+/*
+ *
+ *    Copyright (c) 2020-2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines Base class for a CHIP IM Command
```

Comment:
delete `@file` - tin this case this seems completely wrong and in all cases throghout the codebase, I have not seen these comments be useful. In this case I expect this would say `Contains utilities for cluster objects` which just wastes vertical space in the file.